Data/HashMap/Strict/Base.hs

```diff
@@ -276,11 +281,12 @@ alter f k m =
 {-# INLINABLE alter #-}
 
 -- | /O(log n)/  The expression (@'alterF' f k map@) alters the value @x@ at
--- @k@, or absence thereof. @alterF@ can be used to insert, delete, or update
--- a value in a map.
+-- @k@, or absence thereof.
+--
+-- 'alterF' can be used to insert, delete, or update a value in a map.
 --
 -- Note: 'alterF' is a flipped version of the 'at' combinator from
--- <https://hackage.haskell.org/package/lens-4.15.4/docs/Control-Lens-At.html#v:at Control.Lens.At>.
+-- <https://hackage.haskell.org/package/lens/docs/Control-Lens-At.html#v:at Control.Lens.At>.
```

Comment:
What's the convention for inter-package links? If `at` were to move unexpectedly to a different module or package, that would invalidate a non-versioned link. OTOH, linking to some random old version of `lens` seems pretty silly!

Reply:
I'm not aware of any convention. Frankly, I don't expect `at` to move very soon. If it moves and we then get a report regarding the broken link, that would also seem like a good thing to me.